spec/classes/init_spec.rb

```diff
@@ -107,6 +107,41 @@
       it { expect { should compile }.to raise_error(/is not a boolean/) }
     end
 
+    context 'specific to Redhat 6' do
+      let(:facts) { {
+        :operatingsystem           => 'RedHat',
+        :osfamily                  => 'RedHat',
+        :operatingsystemmajrelease => '6'
+      } }
+
+      it { should contain_file('marathon-conf').with({
+        :ensure => 'file',
+        :path   => '/etc/init.d/marathon'
+      }) }
+
+      context 'master and zk servers with defaults params' do
+        it 'should configure marathon with the correct arguments' do
+          should contain_file('marathon-conf').with_content(/args="--master zk:\/\/localhost:2181\/mesos --zk zk:\/\/localhost:2181\/marathon/ "/)
```

Comment:
shouldn't this be 
``/args="--master zk:\/\/localhost:2181\/mesos --zk zk:\/\/localhost:2181\/marathon "/`` ?